### PR TITLE
Partially fix some minor stuff, and """constructive""" critism that may get passed off as spam.

### PR DESCRIPTION
**Pull request by PF94**
So, I was looking through the code of retrosite and noticed some flaws.

A lot of unused assets and unlisted subsites.

Retrosite (604 MB as of 10/21/2020 without .git folder) is more bloated than 2003page (100 MB as of 10/21/2020 without .git folder)

I scanned the directory with Windirstat and found out that there was 4 duplicate "pidgin 2.11" zip files (which also used to be on 2003page) from Jarhead's subsite, in the "previous sites" section. There is a bunch of zip and MP3 files that are (presumably copyrighted) background music from WeatherStar 4k.

The halloween music is almost 40 mb in size, and isn't even played on every browsers.

TL:DR too many bloat files.

I made a bunch of unrelated minor commits, that mainly just list leo's (broken) subsite and fix the 88x31 2003page button, that is all.

### CODE REVIEW

**Comment by PF94** [comment]
closing this because it's been ignored.

fine, make people think that 2003page died.

fine, make subsites unlisted.

fine, just TALK SHIT ABOUT ME ON 4CHAN OR ESCARGOT DISCORD I DON'T EVEN FUCKING KNOW

**Review by forkiesassds — APPROVED** [review]
Looks good.

**Comment by PF94** [comment]
ty icty

but it looks like xpuser doesn't give a shit about this

**Comment by PF94** [comment]
fucking finally
